python/lsst/ip/isr/overscan.py

```diff
@@ -197,14 +204,26 @@ def run(self, exposure, amp, isTransposed=False):
             parallelOverscanBBox = amp.getRawParallelOverscanBBox()
             imageBBox = amp.getRawDataBBox()
 
-            parallelResults = self.correctOverscan(exposure, amp,
-                                                   imageBBox, parallelOverscanBBox,
-                                                   isTransposed=not isTransposed)
-
-            overscanMean = (overscanMean, parallelResults.overscanMean)
-            overscanSigma = (overscanSigma, parallelResults.overscanSigma)
-            residualMean = (residualMean, parallelResults.overscanMeanResidual)
-            residualSigma = (residualSigma, parallelResults.overscanSigmaResidual)
+            maskIm = exposure.getMaskedImage()
+            maskIm = maskIm.Factory(maskIm, parallelOverscanBBox)
+            makeThresholdMask(maskIm, threshold=self.config.numSigmaClip, growFootprints=0)
+            maskPix = countMaskedPixels(maskIm, self.config.maskPlanes)
+            xSize, ySize = parallelOverscanBBox.getDimensions()
+            if maskPix > xSize*ySize*self.config.maskPixFracThresh:
```

Comment:
Just to be clear, this is a check on the _total_ parallel overscan region, and is separate from any particular check on a given column, which presumably happens in the `correctOverscan` code?  If so, the documentation string on the config field should maybe be "total overscan region" or something.